lib/typescript/render/shared.ts

```diff
@@ -2,19 +2,35 @@ import {isFromContact, Message, Contact} from 'httpclient';
 import {formatTime} from 'dates';
 import {DefaultMessageRenderingProps} from './components';
 
+export interface Command {
+  type: string;
+}
+
+export interface SuggestedReplyCommand extends Command {
```

Comment:
Nice, this is how I imagined it. However don't you think that `SuggestedReply` is a bit too source specific? :) Perhaps we can break it down to "send message" with a generic payload.

Reply:
I think I will keep it this specific till we find another usecase, then we can open it up to be more generic if we understand more about the data 😉